### ISSUE TITLE
Can't navigate through clr-tabs with keyboard Arrows

### ISSUE THREAD

**Issue by iivanovgit** [vmware/clarity#5369]
## Describe the bug
Can't navigate through clr-tabs with keyboard Arrows.

On initial load, when we set the focus on the active tab and try to navigate with keyboard arrows to focus the next one we receive an error which prevents any farther focus actions by receiving the following error:

ERROR TypeError: Cannot read property 'nativeElement' of undefined
    at ClrKeyFocus.get (key-focus.ts:78)
    at ClrKeyFocus.handleKeyboardEvent (key-focus.ts:107)
    at ClrKeyFocus_keydown_HostBindingHandler (key-focus.ts:29)
    at executeListenerWithErrorHandling (core.umd.js? [sm]:14990)
    at wrapListenerIn_markDirtyAndPreventDefault (core.umd.js? [sm]:15025)
    at HTMLUListElement.eval (platform-browser.umd.js? [sm]:877)
    at ZoneDelegate.invokeTask (zone.js:421)
    at Object.onInvokeTask (core.umd.js? [sm]:27820)
    at ZoneDelegate.invokeTask (zone.js:420)
    at Zone.runTask (zone.js:188)

## How to reproduce

1.  Go to 'https://stackblitz.com/edit/clarity-v4-light-theme-tabsfocus'
2.  press "TAB" key on the keyboard so we set the focus on clr-tab "A"
3.  press "ARROW " key on the keyboard
4.  See error on DevTools

## Expected behavior

We should be able to navigate through tabs with ARROW key.

## Versions

**App**

- Angular: [10.0.6]
- Clarity: [4.0.0]

**Device:**

- Type: [PC]
- OS: [Windows10]
- Browser [chrome]
- Version [87.0.4280.88]

## Additional notes

After initial load, if we mouse click to activate another tab and navigate the focus with a keyboard arrow all works as expected.

**Comment by bdryanovski** [comment]
@iivanovgit thanks for reporting this we gonna take a look at it, it seems like a bug but more investigation is needed. Taking into consideration that there are a lot of holidays this days it could take some time.

**Comment by bbogdanov** [comment]
Hi @iivanovgit,

Still investigating the root cause of this issue but found a workaround to get this working. 

[Expected behavior example](https://stackblitz.com/edit/clarity-v4-light-theme-tabsfocus-tgqrjh?file=src/app/tabs/tabs.component.ts) 

Lines 34 and 57. 

For some reason the `currentItem` in `ClrKeyFocus` is `undefined` when the page is loaded.

I'll update this issue when I have more info.

**Comment by Jinnie** [comment]
Looking at the stackblitz example, this is duplication of the "Integration of tabs with the router" issue:
https://github.com/vmware/clarity/issues/627

**Comment by gnomeontherun** [comment]
Closing as a duplicate of #627, please review it for details.
